qui/tray/updates.py

```diff
@@ -87,19 +88,21 @@ def setup_menu(self):
         self.tray_menu.set_reserve_toggle_size(False)
 
         if self.vms_needing_update:
-            self.tray_menu.append(TextItem(_("<b>Qube updates available!</b>")))
+            self.tray_menu.append(TextItem("Qube updates available!"))
```

Comment:
Better apply `_()` to the string constant (tools extracting strings for translation will be happier).